client/core/core.go

```diff
@@ -1959,66 +1960,25 @@ func (c *Core) ReconfigureWallet(appPW, newWalletPW []byte, assetID uint32, cfg
 	// If there are active trades, make sure they can be settled by the
 	// keys held within the new wallet.
 	sameWallet := func() error {
-		ownsAddr := func(addr string) error {
-			owns, err := wallet.OwnsAddress(addr)
-			if err != nil {
-				return err
-			}
-			if !owns {
-				return fmt.Errorf("new wallet does not own address found in active trades: %v", addr)
-			}
-			return nil
-		}
+		hasActiveTrade := false
 		for _, dc := range c.dexConnections() {
-			maybeDifferentWallet := false
 			for _, trade := range dc.trackedTrades() {
 				if !trade.isActive() {
 					continue
 				}
```

Comment:
I think you wanted to get rid of the `dc.trackedTrades` loop entirely, right?

Reply:
Is it correct now?

Reply:
Yup pretty sure.  hasActiveAssetOrders loops through for us